contents/tutorials/how-to-rename-events.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: How to rename events
+sidebar: Docs
+showTitle: true
+author:
+  - bijan-boustani
+date: 2024-10-22
+tags:
+  - actions
+  - events
+---
+
+**Estimated reading time:** 3 minutes ☕☕
+
+**tl;dr** Use [actions](/docs/data/actions). Start capturing events with the new name, and then create an action to group it with the existing event.
+
+---
+
+After you've started capturing events, you might decide that the original name you've given an event doesn't quite suit it. For example, you might have an event called `register_button_clicked` and you want to rename it to `signup_button_clicked`.
+
+However, it's not possible to rename events in PostHog. Instead, you should use [actions](/docs/data/actions) to group the old and new events together. This tutorial shows you how.
+
+## How to create an action
+
+Actions are a way of combining several related events into one. You can match both the old and new event name and get a smooth transition in your insights, without losing historical data.
+
+Start by clicking the **New action** button in the [actions tab](https://us.posthog.com/data-management/actions) in your PostHog dashboard. Then, select the **From event or pageview** option.
+
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/create_first_action_light_63d4f8aa42.png"
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/create_first_action_dark_bb2ec7a894.png"
+    alt="Create an action"
+    classes="rounded"
+/>
+
+Now you can take the two separate events (`register_button_clicked` and `signup_button_clicked`) and combine them. Match groups allow you to find both of those events and see them as one `Sign Up CTA Clicked` action.
+
+In **Match Group #1**, click **Other events** and select the old event name. Then click **+ Add match group** and add the new event name. Then, give your action a name and press the **Save** button.
+
+<ProductVideo
```

Comment:
Video looks good! For future videos, dont forget to use the posthog preset. I wouldnt both changing this now though